src/style.css

```
@@ -1 +1,53 @@
-/* Styles go here */
+.display {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.5);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  justify-content: end;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  gap: 10px;
+}
+
+.nav_link {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.nav_link:active::after {
```

```suggestion
.nav_link::after {
```
@marina-tilniak выше описала что надо это исправить, если тебе не понятны комментарии - спрашивай в слек что именно от тебя хочет ментор